component_sdk/python/kfp_component/google/storage/_download_blob.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os
+
+from google.cloud import storage
+from ._common_ops import parse_blob_path
+
+def download_blob(source_blob_path, destination_file_path):
+    """Downloads a blob from the bucket.
+    
+    Args:
+        source_blob_path (str): the source blob path to download from.
+        destination_file_path (str): the local file path to download to.
+    """
+    bucket_name, blob_name = parse_blob_path(source_blob_path)
+    storage_client = storage.Client()
```

Comment:
Can we add a comment here to instruct the auth config, which I believe is GOOGLE_APPLICATION_CREDENTIALS
https://cloud.google.com/storage/docs/reference/libraries#client-libraries-install-python
This would be useful for the utility user what to expect for the storage.Client() to work.

Reply:
This is required for all google sdks to work. I am going to include it in the GCP component sample notebooks. WDYT?

Reply:
That sounds good.